app/libs/js/LEAF/formGrid.js

```diff
@@ -61,6 +60,107 @@ var LeafFormGrid = function (containerID, options) {
     showIndex = false;
   }
 
+  /**
+   * @param values (required) object of cells and names to generate grid
+   * @param showScriptTags (default false) whether to display script tags
+   * @memberOf LeafFormGrid
+   * Returns copy of values with cells property html entities decoded
+   */
+  function decodeCellHTMLEntities(values, showScriptTags = false) {
+    let gridInfo = { ...values };
+    if (gridInfo?.cells) {
+      let cells = gridInfo.cells.slice();
+      cells.forEach((arrRowVals, ci) => {
+        arrRowVals = arrRowVals.map((v) => {
+          v = v.replaceAll("<", "&lt;"); //handle old data values
+          v = v.replaceAll(">", "&gt;");
+          let elDiv = document.createElement("div");
+          elDiv.innerHTML = v;
+          let text = elDiv.innerText;
+          if (showScriptTags !== true)
+            text = text.replaceAll(
+              /(<script[\s\S]*?>)|(<\/script[\s\S]*?>)/gi,
+              ""
+            );
+          return text;
```

Comment:
This almost caught me.  Looked like you had two different returns for the same function.  Then I remembered you had called the map function above.  ;-)